packages/access-api/src/types/provisions.ts

```diff
@@ -0,0 +1,30 @@
+import * as Ucanto from '@ucanto/interface'
+
+export type AlphaStorageProvider = 'did:web:web3.storage:providers:w3up-alpha'
+
+/**
+ * action which results in provisionment of a space consuming a storage provider
+ */
+export interface StorageProvisionCreation {
+  space: Ucanto.DID<'key'>
+  account: Ucanto.DID<'mailto'>
+  provider: AlphaStorageProvider
+}
+
+/**
+ * stores instances of a storage provider being consumed by a consumer
+ */
+export interface Provisions {
```

Comment:
Nit: I would also find it more intuitive if it was called following instead

```suggestion
export interface ProvisionStore {
```

Reply:
I called Delegations `DelegationsStorage`. When I did alan commented that it didn't look like the other types that were called e.g. `Delegations`. I'm not against changing these names in subsequent PRs. If we call it ProvisionStore we should also change DelegationsStorage to DelegationsStore IMO

Reply:
renamed to ProvisionsStorage and idc if we rename to something else after. https://github.com/web3-storage/w3protocol/pull/462/commits/9b9a14d1d610c4c76860c1bdab14e05956db38f0

(My original intention was to eventually have like `type Provision = StorageProvision | OtherKindOfProvision`)

Reply:
I agree with @gozala but I want consistency in the code base. Can you please open another PR to bring the others in line?